api/units.go

```diff
@@ -76,6 +76,7 @@ func (ur *unitsResource) set(rw http.ResponseWriter, req *http.Request, item str
 	}
 
 	var su schema.Unit
+	newUnit := false
```

Comment:
Not sure why it's better ? if it's just "golang allows it", then for me it's better to have all variables in same place... and this function is not that long to justify this... anyway this is not a big deal, unless you show some optimized generated code or some other official documentation?!